### PR TITLE
feat: add a link to the explorer

### LINKED ISSUE #1339: Show a link to the explorer once the collection is published and has minted items
We might need to check that the item is indeed minted to the current connected address

### LINKED ISSUE #1426: Add 'see in the world' option at collection menu
The link should be: `https://play.decentraland.zone?WITH_COLLECTIONS={URN}`

Should be always **.zone**

### PR DESCRIPTION
**Pull request by nicosantangelo**
Closes #1339
Closes #1426 
Related #1310

### CODE REVIEW

File: .ci/.env.prd [hunk 1]
```diff
@@ -28,7 +28,7 @@ REACT_APP_CONTROLLER_ADDRESS=0x6843291bd86857d97f0d269e698939fb10d60772
 REACT_APP_REGISTRAR_ADDRESS=0x2a187453064356c898cae034eaed119e1663acb8
 REACT_APP_MARKETPLACE_GRAPH_URL=https://api.thegraph.com/subgraphs/name/decentraland/marketplace
 REACT_APP_IPFS_URL=https://ipfs.infura.io:5001/api/v0/add?pin=false
-REACT_APP_LAND_POSITION_URL=https://play.decentraland.org?position=
+REACT_APP_EXPLORER_URL=https://play.decentraland.org
```
Comment (nachomazzara) [inline .ci/.env.prd:31]: Must be `.zone ` for the `WITH_COLLECTION` thing

File: package-lock.json [hunk 2]
```diff
@@ -12024,7 +10644,7 @@
     },
     "node_modules/ethereumjs-abi": {
       "version": "0.6.8",
-      "resolved": "git+https://git@github.com/ethereumjs/ethereumjs-abi.git#ee3994657fa7a427238e6ba92a84d0b529bbcde0",
+      "resolved": "git+ssh://git@github.com/ethereumjs/ethereumjs-abi.git#ee3994657fa7a427238e6ba92a84d0b529bbcde0",
```
Comment (nachomazzara) [inline package-lock.json:10647]: Heads up here @nicosantangelo. IDKW every time you bump dcl-dapps this is changed and gitlab stop working. Can you just change `+ssh` to `+https` I always re-install or change it manually.

File: package-lock.json [hunk 3]
```diff
@@ -36601,7 +34846,7 @@
       }
     },
     "ethereumjs-abi": {
-      "version": "git+https://git@github.com/ethereumjs/ethereumjs-abi.git#ee3994657fa7a427238e6ba92a84d0b529bbcde0",
+      "version": "git+ssh://git@github.com/ethereumjs/ethereumjs-abi.git#ee3994657fa7a427238e6ba92a84d0b529bbcde0",
```
Comment (nachomazzara) [inline package-lock.json:34849]: Same here

File: package.json [hunk 4]
```diff
@@ -36,7 +36,7 @@
     "decentraland-ad": "^1.4.2",
     "decentraland-builder-scripts": "^0.24.0",
     "decentraland-commons": "^5.1.0",
-    "decentraland-dapps": "^12.9.0",
+    "decentraland-dapps": "^12.10.0",
```
Comment (nachomazzara) [inline package.json:39]: there is a new version of dcl-dapps in case you want to bump it

File: src/modules/translation/languages/en.json [hunk 5]
```diff
@@ -921,6 +921,7 @@
     "add_existing_item": "Add Existing Item",
     "forum_post": "Forum post",
     "copy_address": "Copy address",
+    "jump_in": "Jump In",
```
Comment (nachomazzara) [inline src/modules/translation/languages/en.json:924]: Can we use `See in world` or `Test in world` ?

File: src/components/CollectionDetailPage/CollectionMenu/CollectionMenu.tsx [hunk 6]
```diff
@@ -125,6 +125,8 @@ export default class CollectionMenu extends React.PureComponent<Props> {
             inverted
             flowing
           />
+
+          {isCommitteeMember && collection.isPublished ? <Dropdown.Item text={t('collection_menu.jump_in')} /> : null}
```
Comment (nachomazzara) [inline src/components/CollectionDetailPage/CollectionMenu/CollectionMenu.tsx:129]: Based on what we talked about offline, we will allow this also if the collection is not published.

Comment (nachomazzara) [reply src/components/CollectionDetailPage/CollectionMenu/CollectionMenu.tsx:129]: If it is the owner of the collection or manager can also see this button. Remember that the committee has no access to this view.

Comment (nachomazzara) [reply src/components/CollectionDetailPage/CollectionMenu/CollectionMenu.tsx:129]: This button does nothing after being clicked

**Review by nachomazzara — CHANGES_REQUESTED** [review]
Excellent!

File: .ci/.env.prd (hunk 1 above)
Comment (nicosantangelo) [reply .ci/.env.prd:31]: It's being handled here, so we don't have to have two env vars: https://github.com/decentraland/builder/pull/1429/files#diff-294880f7daea062fae2add927da5e496ac51f396bd89cd960673f9da05e3912cR23

**Pull request merged**